mLRS/Common/esp-lib/esp-spi.h

```diff
@@ -14,39 +14,26 @@
 
 //-- select functions
 
-#ifndef SPI_SELECT_PRE_DELAY
-  #define SPI_SELECT_PRE_DELAY
-#endif
-#ifndef SPI_SELECT_POST_DELAY
-  #define SPI_SELECT_POST_DELAY
-#endif
-#ifndef SPI_DESELECT_PRE_DELAY
-  #define SPI_DESELECT_PRE_DELAY
-#endif
-#ifndef SPI_DESELECT_POST_DELAY
-  #define SPI_DESELECT_POST_DELAY
-#endif
-
-
-#ifdef SPI_CS_IO
```

Comment:
pl keep that
pl rename your pin to SPI_CS_IO

Reply:
Added back